entity-types/dpo-olin_dataset_versions/definition.yml

```diff
@@ -0,0 +1,40 @@
+domain: DPO
+type: OLIN_DATASET_VERSIONS
+synthesis:
+  rules:
+  - compositeIdentifier:
+        separator: ":"
+        attributes:
+          - dataset.namespace
+          - dataset.name
+          - updatedAt
```

Comment:
why updatedAt?

Wouldn't this create a new entity every time the field is updated? is that desired?
Can we talk about how many entities are we expecting here?